### ISSUE TITLE
Problems with Vault

### ISSUE THREAD

**Issue by Mihanoks** [MilkBowl/VaultAPI#150]
I'm new to plugin development and I'm trying to create my first plugin. I have a problem with using economic methods. Please help
Code: https://github.com/Mihanoks/WorldBank
Error: [latest.log](https://github.com/MilkBowl/VaultAPI/files/10201031/latest.log)

**Comment by Geolykt** [comment]
A) This is not really a support forum
B) That repository does not seem to be public
C) Guessing from the error message there are a few possible causes:
 ~~1. You are programming an economy **provider** but did not register your economy to vault (or at least not at that point)
 2. You are programming an economy **consumer** but there is no economy **provider** present~~
 3. You are programming an economy **consumer** but all economy **providers** are loading **after** your plugin; Consider using the [ServiceRegisterEvent](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/event/server/ServiceRegisterEvent.html) to determine the currently active economy instead of the code currently present in the documentation.

**Comment by Mihanoks** [comment]
Sorry, I forgot to make it public, now the repository is public

**Comment by Geolykt** [comment]
Yeah then the issue is exactly what I thought it was. You should use the ServiceRegisterEvent to obtain the economy.

Don't ask for examples though, very few plugins actually use that because few know that that event actually exists.
And it's a bit harder to implement I guess

**Comment by creatorfromhell** [comment]
> Yeah then the issue is exactly what I thought it was. You should use the ServiceRegisterEvent to obtain the economy.
> 
> Don't ask for examples though, very few plugins actually use that because few know that that event actually exists. And it's a bit harder to implement I guess

This information is currently in the README on this repository...
